packages/providers/inpage-providers-hub/src/floatingButton/images.tsx

```
@@ -1,22 +1,35 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 
+export function Image({style, ...props}: any) {
+  return (
+    <img
+      style={{
+        userSelect: 'none',
+        WebkitUserDrag: 'none',
+        ...style,
+      }}
+      {...props}
+    />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)